- Fix unused variable old_segv_handler and old_bus_handler

Index: os_dep.c
--- os_dep.c.orig
+++ os_dep.c
@@ -887,7 +887,7 @@ GC_INNER size_t GC_page_size = 0;
 
 #   if defined(SUNOS5SIGS) || defined(IRIX5) || defined(OSF1) \
        || defined(HAIKU) || defined(HURD) || defined(FREEBSD) \
-       || defined(NETBSD)
+       || defined(NETBSD) || defined(OPENBSD)
         static struct sigaction old_segv_act;
 #       if defined(_sigargs) /* !Irix6.x */ || defined(HPUX) \
            || defined(HURD) || defined(NETBSD) || defined(FREEBSD)
